### PR TITLE
Replaced node.js foundation with OpenJS foundation

### PR DESCRIPTION
**Pull request by Relequestual**
If this is not correct, I guess some additional re-phrasing is required.
If so, please close the PR and I'll make an issue in stead, if appropriate.

### CODE REVIEW

**Review by targos — APPROVED** [review]

File: WORKING_GROUPS.md [hunk 1]
```diff
@@ -419,7 +419,7 @@ Responsibilities include:
 * Offer help to npm package maintainers to fix high-impact security bugs.
 * Maintain and make available data on disclosed security vulnerabilities in:
   * the core Node.js project
-  * other projects maintained by the Node.js Foundation technical group
+  * other projects maintained by the OpenJS Foundation technical group
```
Comment (targos) [inline WORKING_GROUPS.md:422]: I don't know if there is such an "OpenJS Foundation technical group"

**Review by mcollina — APPROVED** [review]
lgtm

File: WORKING_GROUPS.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ Note that changes to WG membership require unanimous consensus.  See
 <a id="developers-certificate-of-origin"></a>
 ## Developer's Certificate of Origin 1.1
 
-*Note*: The DCO is mandatory for all Node.js Foundation projects.
+*Note*: The DCO is mandatory for all OpenJS Foundation projects.
```
Comment (ljharb) [inline WORKING_GROUPS.md:195]: I don’t think this is accurate - the OpenJS Foundation recommends but does not require a CLA or a DCO.

Comment (Trott) [reply WORKING_GROUPS.md:195]: Yeah, let's remove this line entirely.

```suggestion
```

File: WORKING_GROUPS.md (hunk 1 above)
Comment (mhdawson) [reply WORKING_GROUPS.md:422]: Looking at the list I think we should just delete everything up to the line "Promote the improvement of security practices within the Node.js ecosystem." as the vulnerability reporting part was dropped a little while back.

Comment (Trott) [reply WORKING_GROUPS.md:422]: We should probably re-charter the group with that removed too.

File: WORKING_GROUPS.md (hunk 2 above)
Comment (Relequestual) [reply WORKING_GROUPS.md:195]: Well, that certinly changes things! It's been a talking point in our project (JSON Schema) and maybe others.

It's part of the project progression checklist: https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboarding-checklist

File: WORKING_GROUPS.md (hunk 1 above)
Comment (Relequestual) [reply WORKING_GROUPS.md:422]: Do you want to create a new issue before resolving this conversation thread @Trott ?

File: WORKING_GROUPS.md (hunk 2 above)
Comment (bnb) [reply WORKING_GROUPS.md:195]: AFAIK it's a "requirement" in that you can do a DCO or CLA, and that there's leniency if implementing such a process is extremely burdensome (like in the case of Node.js, where we seemingly haven't fully done DCO correctly but are aiming at doing it correctly).

Comment (mhdawson) [reply WORKING_GROUPS.md:195]: It is a requirement and it's documented here: https://github.com/openjs-foundation/cross-project-council/blob/main/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla

Comment (mhdawson) [reply WORKING_GROUPS.md:195]: To be clear the requirement is to choose `either` the DCO or CLA so it's not that a DCO is is required but as @bnb mentioned you need to use one of the two.

Comment (Relequestual) [reply WORKING_GROUPS.md:195]: @mhdawson OK, is the action here modify the wording to say "DCO or CLA"?

Comment (mhdawson) [inline WORKING_GROUPS.md:195]:
```suggestion
Use of a CLA or DCO is mandatory for all all OpenJS Foundation projects. The Node.js project has chosen to use the DCO 1.1
```

Comment (mhdawson) [reply WORKING_GROUPS.md:195]: @Relequestual I think my suggestion above might address what we should say.

Comment (Relequestual) [reply WORKING_GROUPS.md]: Resolved by comitting suggested change to this PR

File: WORKING_GROUPS.md (hunk 1 above)
Comment (Relequestual) [reply WORKING_GROUPS.md:422]: I created a new issue for you so I can resolve this comment.
https://github.com/nodejs/TSC/issues/1081

**Comment by Relequestual** [comment]
@mhdawson Looks like this could be merged now? =]
Glad I can contribute in however small a way.

File: WORKING_GROUPS.md (hunk 1 above)
Comment (mhdawson) [inline WORKING_GROUPS.md:422]:
```suggestion 
  * other projects maintained by the overall Node.js project 
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: WORKING_GROUPS.md [hunk 3]
```diff
@@ -419,7 +419,7 @@ Responsibilities include:
 * Offer help to npm package maintainers to fix high-impact security bugs.
 * Maintain and make available data on disclosed security vulnerabilities in:
   * the core Node.js project
-  * other projects maintained by the Node.js Foundation technical group
+  * other projects maintained by the overall Node.js project
```
Comment (mhdawson) [inline WORKING_GROUPS.md:422]:
```suggestion
  * other projects maintained by the overall Node.js project
```

**Comment by mhdawson** [comment]
@Relequestual thanks for your help on this.

**Pull request merged**